### PR TITLE
Fixing the issues #11

### PR DESCRIPTION
**Pull request by FlorianWilhelm**
Trying to fix the issues in #11 as suggested by @abravalheri.

### CODE REVIEW

File: tests/test_custom_extension.py [hunk 1]
```diff
@@ -74,3 +76,26 @@ def test_generated_extension(tmpfolder):
     with chdir("my_project"):
         # Testing a project generated by the custom extension
         run_common_tasks(tests=False)
+
+
+def test_new_project_does_not_fail_pre_commit(cwd, pre_commit, putup):
+    # Given pyscaffold is installed,
+    # when we call putup with extensions and pre-commit
+    name = "my_project"
+    run(f"{putup} --pre-commit --dsproject -p my_package {name}")
```
Comment (abravalheri) [inline tests/test_custom_extension.py:85]: Should we test using a `--namespace my.pkg` here so we ensure the worst case scenario is working?

**Review by abravalheri — APPROVED** [review]
Thank you very much @FlorianWilhelm, that was super quick!

File: tests/test_custom_extension.py (hunk 1 above)
Comment (FlorianWilhelm) [reply tests/test_custom_extension.py]: Good point! I fixed this in the last commit.

**Comment by abravalheri** [comment]
Great @FlorianWilhelm, it looks very good!

**Pull request merged**